packages/matchbox/src/components/Label/Label.js

```diff
@@ -1,21 +1,38 @@
 import React from 'react';
-import classnames from 'classnames';
-import styles from './Label.module.scss';
+import { Box } from '../Box';
+import { ScreenReaderOnly } from '../ScreenReaderOnly';
 
-const Label = ({
-  label,
-  id,
-  className,
-  children
-}) => (
-  <label
-    id={id && `${id}Label`}
-    htmlFor={id}
-    className={classnames(styles.Label, className)} >
-    {label}
-    {children}
-  </label>
-);
+function Label(props) {
+  const { label, id, className, children, labelHidden } = props;
+
+  if (labelHidden) {
+    return (
+      <ScreenReaderOnly>
+        <Box as="label" id={id && `${id}Label`} htmlFor={id}>
+          {label}
+          {children}
+        </Box>
+      </ScreenReaderOnly>
+    );
+  }
```

Comment:
I don't think labelHidden was supported previously with Select, so this would be a new prop. At least, I can't find any selects with labelHidden

https://github.com/SparkPost/2web2ui/search?q=labelHidden&unscoped_q=labelHidden

![Screen Shot 2020-03-04 at 9 31 23 AM](https://user-images.githubusercontent.com/3903325/75889490-05944900-5dfb-11ea-9be7-e503dd54ea37.png)

Reply:
Right - yeah I think we shouldn't incorporate this yet otherwise we could have backwards/forwards compatibility problems.

Reply:
Hm I don't think I understand the concern. If it isn't being used currently, it wont have any effect on prehibana components right? But we do have to make sure not to use it until q3?

Reply:
Yeeeah exactly - so we passed in a hidden label for the Hibana version, but the old version has a hidden label already as a separate component that would be tricky to handle

Reply:
Gotcha, I'll leave the labelHidden prop on Label for Textfield and the other components that use it, but will comment out the prop in Select for now